app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultFragment.java

```diff
@@ -30,9 +38,17 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
             @Nullable Bundle savedInstanceState) {
         List<Product> products = new ArrayList<>();
         ProductSearchResultAdapter adapter = new ProductSearchResultAdapter(getContext(), products);
-        mViewModel = new ProductSearchResultViewModel(adapter);
+        Navigator navigator = new Navigator(getParentFragment().getParentFragment());
+        mViewModel = new ProductSearchResultViewModel(adapter, navigator);
+
+        RealmApi realmApi = new RealmApi();
+        SharedPrefsApi prefsApi = new SharedPrefsImpl(getActivity());
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to prefsApi in com.framgia.forder.screen.searchproduct.ProductSearchResultFragment.onCreateView(LayoutInflater, ViewGroup, Bundle)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE